app/views/posts/new.html.erb

```diff
@@ -1,6 +1,5 @@
 <%= form_for @post do |form| %>
```

Comment:
Any reason form_with isn't used here?

Reply:
form_with join the two previous form_for and form_tag and is a new feature of rails 5.1.
If we want to make our code uniform we need to decide whether to change all the form_form with form_with.